README.md

```diff
@@ -560,9 +615,147 @@ module.exports = buildWebpackConfig([
 If you just need the app to work offline, use the [offline-plugin](https://github.com/NekR/offline-plugin).
 
 Otherwise, put the `service-worker.js` file in the `public/` folder, and register it normally.
+
+</details>
+
+<details>
+<summary>I need title and meta tags for each route for SEO. How do I do it?</summary>
+
+You can use [`react-helmet`](https://github.com/nfl/react-helmet) to dynamically add html tags to the `<head>` of a document and [`react-snap`](https://github.com/stereobooster/react-snap) to prerender them statically after the build process is complete. Here's how to configure them.
+
+## Install `react-helmet` and `react-snap`
+
+`yarn add react-helmet react-snap`
+
+## Add meta tags for each route in the `render` function
+
+As specified in the [`react-helmet` documentation](https://github.com/nfl/react-helmet). E.g.
+
+```
+export default class Home extends React.Component {
+  render() {
+    return (
+      <Helmet>
+        <title>Home title</title>
+        <meta name="robots" content="index,follow" />
```

Comment:
if you want SEO, why do you prohibit indexing?

Reply:
Hi stereobooster, thank you for taking the time to make comments on this pr, very appreciated, and thanks for the amazing work with react-snap. I agree this is redundant and will be removed.